docs/dapp/sapphire/gsn.md

```diff
@@ -0,0 +1,76 @@
+---
+description: CLI instructions for deploying GSN on Sapphire Testnet
+---
+# Gas Station Network
+
+[Gas Station Network](https://docs.opengsn.org) is very useful for improving user experiences of many Dapps by making their users send transactions without having native tokens for gas. We make GSN functionalities available on Sapphire Paratime.
+
+### Package Install
+
+Starting with an empty folder, let us install the GSN command line tool by using the following commands:
+
+```shell
+npm init -y
+npm install --save-dev @oasislabs/opengsn-cli 
+```
+
+### Deploy GSN Contracts
+
+We can deploy GSN contracts by the following command. Remember to replace with your own private key and account address in it.
+
+```shell
+npx gsn deploy --network sapphire_testnet --burnAddress 0xfA3AC9f65C9D75EE3978ab76c6a1105f03156204 --devAddress 0xfA3AC9f65C9D75EE3978ab76c6a1105f03156204 --testToken true --testPaymaster true --yes --privateKeyHex '8a61cab2fd89c40c0f87275f2547bf0139fe78c4aa37eed0ee4357ce88033aae'
+```
+
+After the command finishes successfully, you should find the addreses of deployed contracts at the end:
+
+```shell
```

Comment:
`shell` block type should only be used for commands that can be copied&pasted directly into the command line.

```suggestion
```
```